frontend/src/scenes/instance/AsyncMigrations/AsyncMigrations.tsx

```diff
@@ -243,11 +275,26 @@ export function AsyncMigrations(): JSX.Element {
                     />
 
                     <Tabs activeKey={activeTab} onChange={(t) => setActiveTab(t as AsyncMigrationsTab)}>
-                        <TabPane tab="Management" key={AsyncMigrationsTab.Management} />
+                        <TabPane
+                            tab={`Management (${actionableMigrations.length})`}
+                            key={AsyncMigrationsTab.Management}
+                        />
+                        <TabPane
```

Comment:
This tab isn't relevant in 99% of the cases. This should be hidden by default.

Reply:
good idea to hide it by default, but I'm not that good at frontend stuff to know how to that fast